package.json

```diff
@@ -3,7 +3,7 @@
   "description": "The Internet Archive Collection Browser.",
   "license": "AGPL-3.0-only",
   "author": "Internet Archive",
-  "version": "2.7.0",
+  "version": "2.7.1-alpha.9",
```

Comment:
Just a reminder not to merge in the alpha version!

Reply:
Thank you!!